test/integration/filters/array.spec.ts

```diff
@@ -417,6 +417,32 @@ describe('filters/array', function () {
         - Boring sneakers
```

Comment:
great job! seems to be backward compatible, at least for the existing test cases.